lms/static/scripts/frontend_apps/components/dashboard/test/PaginatedMultiSelect-test.js

```diff
@@ -108,16 +108,27 @@ describe('PaginatedMultiSelect', () => {
     it('loads next page when scroll is at the bottom', () => {
       const { select, fakeLoadNextPage } = getScrollableSelect();
 
-      scrollTo(select, 160);
+      scrollTo(select, { scrollHeight: 160 });
       assert.called(fakeLoadNextPage);
     });
 
     it('does nothing when scroll is not at the bottom', () => {
       const { select, fakeLoadNextPage } = getScrollableSelect();
 
-      scrollTo(select, 250);
+      scrollTo(select, { scrollHeight: 250 });
       assert.notCalled(fakeLoadNextPage);
     });
+
+    it('does not scroll again if scrolling up', () => {
+      const { select, fakeLoadNextPage } = getScrollableSelect();
+
+      // We scroll down, then a little bit up, still inside the offset gap
+      scrollTo(select, { scrollHeight: 160, scrollTop: 155 });
+      scrollTo(select, { scrollHeight: 160, scrollTop: 150 });
+
+      // The page is attempted to load only once, and ignored when scrolling up
+      assert.calledOnce(fakeLoadNextPage);
```

Comment:
This test fails in `main` branch, and reports the stub was called twice instead of once.